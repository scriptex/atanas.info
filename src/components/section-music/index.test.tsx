import * as React from 'react';
import { render } from '@testing-library/react';

import { SectionMusic } from '.';

Object.defineProperty(global.window.HTMLMediaElement.prototype, 'load', {
	configurable: true,
	get() {
		return () => undefined;
	}
});

describe('SectionMusic component', () => {
	it('Should render the SectionMusic component', () => {
		const { asFragment } = render(<SectionMusic />);

		expect(asFragment()).toMatchSnapshot();
	});
});
