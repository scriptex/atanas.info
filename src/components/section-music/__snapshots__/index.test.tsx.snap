// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SectionMusic component Should render the SectionMusic component 1`] = `""`;
